recipes/tracy/all/conanfile.py

```diff
@@ -0,0 +1,95 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.43.0"
+
+
+class TracyConan(ConanFile):
+    name = "tracy"
+    description = "C++ frame profiler"
+    topics = ("tracy", "profiler", "performance", "gamedev")
```

Comment:
```suggestion
    topics = ("profiler", "performance", "gamedev")
```

Don't need to add project's name as topic. It should help to find related tags to the project name.

Reply:
Fixed